mappings/servers/craftok/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "craftok",
+    "name": "Craftok",
+    "description": "The French minigames server",
+    "addresses": [
+        "craftok.fr"
+    ],
+    "primaryAddress": "play.craftok.fr",
+    "minecraftVersions": [
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#55FFFF",
+    "secondaryColor": "#FF5555",
+    "primaryLanguage": "fr",
+    "languages": [
+        "fr"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU"
+    ],
+    "gameTypes": [
+        "Minigames",
+        "PvP",
+        "Survival"
+    ],
+    "website": "https://craftok.fr/",
+    "store": "https://store.craftok.fr/",
```

Comment:
You used the wrong URL here.
The right one is https://shop.craftok.fr/

Reply:
mb